lib/jekyll/algolia/indexer.rb

```diff
@@ -296,6 +296,8 @@ def self.remote_settings
       # If the settingID are not matching, it means our config is different, so
       # we push it, overriding the settingID for next push.
       def self.update_settings
+        return {} if Configurator.settings.empty?
```

Comment:
I think here there is no return value needed

Reply:
Agreed, `return if Configurator.settings.empty?` should be enough. We just want to fail fast here, we don't care about the return value

Reply:
```suggestion
        return if Configurator.settings.empty?
```